packages/nouns-contracts/contracts/interfaces/INounsAuctionHouseV2.sol

```diff
@@ -151,4 +164,8 @@ interface INounsAuctionHouseV2 {
     function duration() external view returns (uint256);
 
     function biddingClient(uint256 nounId) external view returns (uint32 clientId);
+
+    function setPrices(SettlementNoClientId[] memory settlements) external;
+
+    function streamEscrow() external view returns (IStreamEscrow);
```

Comment:
probably want to remove V3 related stuff from this V2 file?

Reply:
yup 04c70a2e17775d78144447d254e4488c9ff7e01c